### DISCUSSION TITLE
Audit logs Piccolo app

### DISCUSSION THREAD

**Discussion by sinisaos** [piccolo-orm/piccolo_api#159] [Ideas]
@dantownsend I don’t want to be intrusive and I'm sorry because I'm constantly opening new PRs that need to be reviewed (few already open ones in the Piccolo API and Piccolo Admin), but I've created a Piccolo app that tracks changes made by authenticated users to database tables (audit logs), for which I think would be useful as an integral part of the Piccolo API (similar to the [session_auth](https://piccolo-api.readthedocs.io/en/latest/session_auth/tables.html) app). If necessary, more columns can be added to the `AuditLog` class. If you want I can do PR so you can see what it’s all about. Here's a video for it (no changes required in Piccolo Admin).

https://user-images.githubusercontent.com/30960668/175811230-1ce8398f-2ef5-4f0d-b66c-8bfb97ff2b18.mp4

**Comment by dantownsend** [comment]
@sinisaos Looks very interesting.

I've worked on projects in the past where clients require this feature, so I can see the value. Feel free to open a PR.
